src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -176,11 +183,30 @@ else if ( commonContigs.size() == dict1.size() && commonContigs.size() == dict2.
             return SequenceDictionaryCompatibility.IDENTICAL;
         else if ( ! commonContigsAreAtSameIndices(commonContigs, dict1, dict2) )
             return SequenceDictionaryCompatibility.DIFFERENT_INDICES;
+        else if ( twoSupersetsOne(commonContigs, dict1, dict2))
+            return SequenceDictionaryCompatibility.TWO_SUPERSETS_ONE;
         else {
             return SequenceDictionaryCompatibility.COMMON_SUBSET;
         }
     }
 
+
+    /**
+     * Utility function that tests whether dict2's set of contigs supersets dict1's
+     *
+     * @param commonContigs
+     * @param dict1
+     * @param dict2
+     * @return true if dict2's set of contigs supersets dict1's
+     */
+    private static boolean twoSupersetsOne(Set<String> commonContigs, SAMSequenceDictionary dict1, SAMSequenceDictionary dict2) {
+        Set<String> nameSetOne = getContigNames(dict1);
+        Set<String> nameSetTwo = getContigNames(dict2);
+        return nameSetTwo.containsAll(nameSetOne);
```

Comment:
This check is not quite sufficient -- you want to make sure that the reference dictionary contains all the contigs from the reads **with the same length for each contig**. Easiest thing to do is to make these a `Set<SAMSequenceRecord>` -- you can get the `SAMSequenceRecords` out of each `SAMSequenceDictionary` via the `getSequences()` method.